per/drf_views.py

```diff
@@ -362,3 +364,12 @@ def get_queryset(self):
                 "-assessment_number").distinct("country_id").values("id")
         )
         return self.get_filtered_queryset(self.request, queryset, dispatch=0)
+
+
+class OpsLearningViewset(viewsets.ModelViewSet):
+    """
+    A simple ViewSet for viewing and editing OpsLearning records.
+    """
+    queryset = OpsLearning.objects.all()
```

Comment:
We also have PublicOpsLearningSearializer, so that means normal/guest can also see them, right?

We probably need to update our permission module for each operation GET/POST/PATCH/DELETE where GET is for everyone with different serializer and POST/PATCH/DELETE is for admin/ops-maintainer.
```python
if request.method in permissions.SAFE_METHODS:
    # Check permissions for read-only request
else:
    # This will be POST/PATCH/DELETE
    # Check permissions for write request
```

To specify which fields are read only, we can use this https://www.django-rest-framework.org/api-guide/serializers/#specifying-read-only-fields